dns/perf/params/default.yaml

```diff
@@ -0,0 +1,15 @@
+# Default parameters that explores performance space given limited CPU.
```

Comment:
I guess we will need to revisit resource limits for this pod at some point; my thought is that DNS is worthy of not being resource constrained.